mypyc/test-data/irbuild-dict.test

```diff
@@ -312,3 +312,14 @@ L0:
     r0 = load_address PyDict_Type
     x = r0
     return 1
+
+[case testDictCopy]
+from typing import Dict
+def f(d: Dict[int, int]) -> Dict[int, int]:
+    return d.copy() # type: ignore
```

Comment:
Can you add `copy()` to the test stubs in ` mypyc/test-data/fixtures/ir.py` and remove `# type: ignore` afterwards? Using `# type: ignore` can be problematic, as it can hide legitimate errors.

Reply:
I remember encountering some issues when removing the `# type: ignore` on CI that did not show up locally, so I ended up adding it as a temporary workaround. Removed it anyways and added `copy()` on `mypyc/test-data/fixtures/ir.py`